src/pages/questions/index.jsx

```diff
@@ -24,12 +24,12 @@ function QuestionsConsumer() {
           <QuestionDisplay />
         </Stack>
       </Grid>
-      <Grid item xs={12} md={5}>
+      <Grid item xs={12} md={6}>
         <ProductInformation />
       </Grid>
-      <Grid item xs={12} md={2}>
+      {/* <Grid item xs={12} md={2}>
         <UserData />
-      </Grid>
+      </Grid> */}
```

Comment:
You moved that, to the top, but it will be problematic, because when user answer questions, it fills with the list of previously answered questions

<img width="723" height="494" alt="image" src="https://github.com/user-attachments/assets/0272b332-6bbc-46a4-9b5a-5a09f4ddda5a" />

If you test plus don't answer randomly since answers impact the database